### PR TITLE
Added pull request template

### PR DESCRIPTION
**Pull request by Aman-Codes**
@priyansh19  Please review my pull request and do let me know if any changes are required.
Added pull_request_template.md
Closes: #233   
### Pull request
pull request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236&bid=17122289)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe):
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+## Issue that this pull request solves
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

Comment (priyansh19) [reply .github/ISSUE_TEMPLATE/pull_request_template.md]: remove extra spaces between "- and [ ]"

**Comment by priyansh19** [comment]
I asked you to directly put this Pull_Request_Template in .github directory but still, you are placing it in Issue_Templates directory
Understand, This is not an issue

**Comment by Aman-Codes** [comment]
@priyansh19 Did the asked changes.

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 10
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236&bid=17122289)

File: .github/Pull_Request_Template.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Documentation update (Documentation content changed)
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Other (please describe):
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

File: .github/Pull_Request_Template.md [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
```
Comment (rpotter12) [inline .github/Pull_Request_Template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222236)

**Pull request merged**